_includes/scripts.html

```diff
@@ -8,3 +8,7 @@
   {% assign _src = _script.src | default: _script %}
 <script src="{{ _src | relative_url }}"></script>
 {% endfor %}{% endfor %}
+{% if page.layout == "help" and page.category != "verify-your-identity" %}
```

Comment:
now that we have `include_touchpoints` I think we can do:

```suggestion
{% if page. include_touchpoints %}
```

Reply:
For some reason, `page.include_touchpoints` did not work. I tried it with `page.include_touchpoints == true` as well and the script would not load 🤔

Reply:
Since the variable is assigned in the layout, it may need to be:

```suggestion
{% if layout.include_touchpoints == true %}
```